Cargo.toml

```diff
@@ -8,7 +8,7 @@ build = "build.rs"
 
 [dependencies]
 wc = { git = "https://github.com/WalletConnect/utils-rs.git", tag = "v0.7.0", features = ["full"] }
-cerberus = { git = "https://github.com/WalletConnect/cerberus.git", tag = "v0.9.1" }
+#cerberus = { git = "https://github.com/WalletConnect/cerberus.git", tag = "v0.9.1" }
```

Comment:
Can we get the [`justfile` update](https://github.com/WalletConnect/blockchain-api/pull/382/files#diff-deb9bb56fb122db0b605aa5b63f95a4665c905b18dd670e1fa6c877576a94ff1R60) in this repo too?